src/test/resources/com/puppycrawl/tools/checkstyle/checks/javadoc/javadocleadingasteriskalign/InputJavadocLeadingAsteriskAlignCorrect.java

```diff
@@ -0,0 +1,94 @@
+/*xml
+<module name="Checker">
+  <module name="TreeWalker">
+    <module name="JavadocLeadingAsteriskAlign">
+      <property name="tabWidth" value="2"/>
+    </module>
+  </module>
+</module>
+*/
+
+package com.puppycrawl.tools.checkstyle.checks.javadoc.javadocleadingasteriskalign;
+
+/** Title
+ * Javadoc for class
+ */
+public class InputJavadocLeadingAsteriskAlignCorrect {
+
+  /** javadoc for instance variable. */
+  private int a;
+
+  /** */
+  private int b;
+
+  /***/
+  private String str;
+
+  /** */
+  private String str1;
+
+  /**
+    This javadoc is allowed because there is no leading asterisk.
+   */
+  public InputJavadocLeadingAsteriskAlignCorrect() {}
+
+  /***
+   * @param a testing....
+   */
+  public InputJavadocLeadingAsteriskAlignCorrect(int a) {}
+
+  /*************************************************
+   *** @param str testing.....
+   **********************************/
+  public InputJavadocLeadingAsteriskAlignCorrect(String str) {}
+
+  /** * */
+  private String str2;
+
+  /****/
+  private String str3;
+
+  /**
+   * This method does nothing.
+   */
+  private void foo() {}
+
+  /**
+   * Javadoc for foo2
+ This is allowed */
+  private void foo2() {
+    // foo2 code goes here
+  } /**
+     * This is also allowed
+     */
+  public void foo3() {}
+
+  /**
+   * Javadoc for enum
+   * */
```

Comment:
Is there no violation here for the `*/` because it doesn't start the line and we validate the prior alone `*` instead? If so this should be clarified in the document.

What if we had a line like ` text */`?

Sounds like we need another check to ensure the `*/` is alone on the line for those who want it.

Reply:
> The documentation examples contains this already:

I meant in written form. You already mentioned about a line not containing a lead. If the ending `*/` is violated in some cases, then I would mention that so it is clear.

Reply:
done. updated the documentation to mention this, here is the new documentation:

https://github.com/checkstyle/checkstyle/blob/0c885e85b4ffc6c87ba4edb32d72f52165b70606/src/main/java/com/puppycrawl/tools/checkstyle/checks/javadoc/JavadocLeadingAsteriskAlignCheck.java#L35-L45

look at the line 41,42,43,44